### ISSUE TITLE
Feature request: Hide Search button

### ISSUE THREAD

**Issue by cconur** [yuliskov/SmartTubeNext#1246]
It would be great to have the possibility to hide the search button (disable it from the interface), in the same way you can now select which section you want to have visible. I'm requesting it to limit the access to not supervised or potentially inappropriate content for kids. Thanks and big congrats for this astonishing app!!

**Comment by netz-filter** [comment]
The microphone button for voice search has no function on the Fire TV and could also be hidden?

**Comment by cconur** [comment]
Someone else to ask for this feature? this would be really useful to avoid kids search for non authorized content. Thanks!

**Comment by user72356** [comment]
I have the same problem, for the same reason. Ridiculous that this has been open for so long. Does anyone know an alternative to this app that allows to do that?

**Comment by rejuce** [comment]
Yes same issue... The app is great, to remove crap like shorts and so children only see allowed channels, but the search button breaks the fence
